policyengine_us/variables/gov/states/id/tax/income/credits/grocery/id_grocery_credit.py

```diff
@@ -4,27 +4,10 @@
 class id_grocery_credit(Variable):
     value_type = float
     entity = TaxUnit
-    label = "Idaho base grocery credit"
+    label = "Idaho grocery credit"
     unit = USD
     definition_period = YEAR
-    defined_for = StateCode.ID
+    defined_for = "id_grocery_credit_eligible"
```

Comment:
eligibility is person-level, so this needs to move to the components. leave this defined for as state